integration_test/azure-pipelines/windows-robot-integration-test.yml

```diff
@@ -18,10 +18,10 @@ jobs:
         Image: 'windows-latest'
```

Comment:
I would suggest making this windows-2022 so that we are always locked in to a version.  
We do this now for mu ci as well.